pages/price-feeds/use-real-time-data/starknet.mdx

```diff
@@ -0,0 +1,210 @@
+---
+description: Consume Pyth Network prices in Starknet applications
+---
+
+import { Callout, Tabs } from "nextra/components";
+
+# How to Use Real-Time Data in Starknet Contracts
+
+This guide explains how to use real-time Pyth data in Starknet contracts.
+
+## Install the Pyth SDK
+
+Use the following dependency in your `Scarb.toml` file to use the latest Pyth Starkenet package:
+
+```toml copy
+[dependencies]
+pyth = { git = "https://github.com/pyth-network/pyth-crosschain.git", tag = "pyth-starknet-contract-v0.1.0"}
+```
+
+Pyth also provides a javascript SDK to interact with the Pyth contract on Starknet. You can install it using the following command:
+
+<Tabs items={["npm", "yarn"]}>
+  <Tabs.Tab>
+    ```sh copy $ npm install --save @pythnetwork/pyth-starknet-js ```
+  </Tabs.Tab>
+  <Tabs.Tab>```sh copy $ yarn add @pythnetwork/pyth-starknet-js ```</Tabs.Tab>
+</Tabs>
+
+## Write Contract Code
+
+The code snippet below provides an example module fetching the ETH/USD price from Pyth price feeds:
+
+```rust {12,17,49,57,62-64} copy
+#[starknet::interface]
+pub trait IFetchPrice<T> {
+    fn example_method(
+        ref self: T, destination: ContractAddress, amount_in_usd: u256, price_update: ByteBuffer
```

Comment:
I suggest adding code comments throughout this example highlighting the important bits. E.g., here, teh important bit is the price_update argument

Reply:
We can add back the comments from https://github.com/pyth-network/pyth-examples/blob/main/price_feeds/starknet/send_usd/contract/src/lib.cairo#L6.